assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -86,6 +86,158 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	describe( "should render an open text question when the `question_type` is 'open_text'", () => {
+		beforeEach( () => {
+			registry
+				.dispatch( CORE_USER )
+				.receiveTriggerSurvey( fixtures.singleQuestionOpenText, {
+					triggerID: 'jestSurvey',
+				} );
+
+			fetchMock.post(
+				/^\/google-site-kit\/v1\/core\/user\/data\/survey-event/,
+				{ body: {}, status: 200 }
+			);
+		} );
+
+		it( 'should  display question text and subtitle correctly', async () => {
+			const { getByText } = render( <CurrentSurvey />, {
+				registry,
+			} );
+
+			// Check question_text is set by question_text prop.
+			expect(
+				getByText( 'How satisfied are you with Site Kit?' )
+			).toBeInTheDocument();
+
+			// Check subtitle is set by subtitle prop.
+			expect(
+				getByText( 'Based on your experience so far, tell us.' )
+			).toBeInTheDocument();
+		} );
+
+		it( 'should limit text input to 100 characters', async () => {
+			const { getByText, getByLabelText, getByDisplayValue } = render(
+				<CurrentSurvey />,
+				{
+					registry,
+				}
+			);
+
+			// Check question_text is set by question_text prop.
+			expect(
+				getByText( 'How satisfied are you with Site Kit?' )
+			).toBeInTheDocument();
+
+			// Check subtitle is set by subtitle prop.
+			expect(
+				getByText( 'Based on your experience so far, tell us.' )
+			).toBeInTheDocument();
```

Comment:
These are tested elsewhere, so can be removed from this test.

Reply:
Ooops! Nice spot :smile: